high_performance/modules/high_performance/include/klepsydra/high_performance/event_loop_middleware_provider.h

```diff
@@ -79,12 +79,12 @@ class EventLoopMiddlewareProvider
                                 const std::string &name = "kpsr_EL",
                                 const long timeoutUS = EVENT_LOOP_START_TIMEOUT_MICROSEC,
                                 const std::vector<int> &cpuAffinity = {},
-                                EventEmitterType eventEmitterType = EventEmitterType::SAFE)
+                                EventEmitterType eventEmitterType = EventEmitterType::UNSAFE_MULTI)
```

Comment:
default should be safe

Reply:
21x replacements.

Reply:
Wrong massive replacement, reverted below.